src/nbl/video/COpenGLShader.h

```diff
@@ -17,26 +17,20 @@ namespace nbl::video
 class COpenGLShader : public IGPUShader
```

Comment:
nuke OpenGL!

Reply:
Oh no, everything?! Can I do it later? I'm scared :))

Reply:
just the implementations of the functions, slap `assert(false)` in them

Reply:
Can you get someone else to do this? I don't want to.